server/utils/notifications/followers-article-mail.util.js

```diff
@@ -0,0 +1,43 @@
+import sendMail from '../mail.util';
+import db from '../../models';
+
+const { User, Notification } = db;
+
+/**
+ * Process new article email notification to followers
+ * @param {int} userId
+ * @param {string} articleName
+ * @returns null
+ */
+
+const newArticleMail = async (userId, articleName) => {
+  const user = await User.findByPk(userId);
+
+  const followers = await user.getFollowers({
+    attributes: ['id', 'email', 'firstname', 'lastname'],
+    where: { subscribed: true }
+  });
+
+  if (followers.length === 0) {
+    return;
+  }
+
+  followers.forEach(async (follower) => {
```

Comment:
In a situation where the followers are so much, how does this affect performance?
Is there a better way to solve for this aside from sending emails in a loop.